src/NUnitFramework/framework/Constraints/EqualConstraint.cs

```diff
@@ -471,6 +491,9 @@ public override string Description
                 if (_comparer.IgnoreWhiteSpace)
                     sb.Append(", ignoring white-space");
 
+                if (_comparer.IgnoreWhiteSpace)
```

Comment:
I guess I'll have to fix this copypasta. I can do this after the first round of reviews.